parsec/core/fs/remote_loader.py

```diff
@@ -16,6 +17,7 @@
     BlockAccess,
     RealmRoleCertificate,
     BaseManifest as BaseRemoteManifest,
+    AnyManifest as RemoteAnyManifest,
```

Comment:
The local one is called `AnyLocalManifest`, so we should call this one `AnyRemoteManifest`

Reply:
will be done in #2865